src/main/java/com/lucidworks/spark/SolrRelation.java

```diff
@@ -31,13 +39,15 @@
   public static String SOLR_QUERY_PARAM = "query";
   public static String SOLR_SPLIT_FIELD_PARAM = "split_field";
   public static String SOLR_SPLITS_PER_SHARD_PARAM = "splits_per_shard";
-
+  public static String PRESERVE_SCHEMA = "preserveschema";
+  protected String preserveSchema;
```

Comment:
let's make this a boolean, so we can parse the property into a true/false in just one place and then just use normal boolean comparisons on it everywhere else. Seems safer.